app/src/main/java/com/github/se/polyfit/ui/viewModel/GraphViewModel.kt

```diff
@@ -101,7 +186,9 @@ private val mockDates =
         LocalDate.of(2024, 3, 17))
 
 fun DateList(): List<LocalDate> {
-  return mockDates
+  val dateList: MutableList<LocalDate> = mutableListOf()
```

Comment:
why are we still using mockData here?

Reply:
it was for the tests but I'll move it to the test class